tcwebhooks-rest-api/src/main/java/webhook/teamcity/server/WebHookTeamCityRestApiZipPluginFixer.java

```diff
@@ -219,7 +229,7 @@ protected boolean doFilesExistInPluginsUnpackedDir(String pluginsDir, String[] f
 		
 		if (restPluginUnpackedDir.exists() && restPluginUnpackedDir.isDirectory()) {
 			for (String filename : filenames) {
-				File jaxbjar = new File(restPluginUnpackedDir + File.separator + filename.replaceAll("/", File.separator));
+				File jaxbjar = new File(restPluginUnpackedDir + File.separator + filename.replace("/", File.separator));
```

Comment:
Why does replaceAll not work here? Just curious

Reply:
Because replaceAll expects a regex. On windows paths may include \ which is parsed as an escape sequence. Replace does the same thing without regex

Reply:
Thanks. That makes sense.